spec/controllers/admin/users_controller_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+require 'spec_helper'
+
+describe Admin::UsersController, type: :controller do
```

Comment:
We infer spec type from file location - shouldn't need to tag this with spec type